cognitedata/reveal
# feat(react-components): add fetching of the view for mapped equipment hook

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:

Related to 3D search panel in FDX.

## Description :pencil:


In order to display 3D mapped only data in search panel in FDX app the hook for getting mapped equipment is required to supply view of corresponding equipment. This PR adds this capability together with small fixes like supporting pagination for instance inspection query and using cache for camera navigation hook.

## How has this been tested? :mag:



In HighlightNode story 


## Test instructions :information_source:


Go to HighlightNode story and observe that everything is styled in a correct way and with decent performance.

## Checklist :ballot_box_with_check:
